lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -0,0 +1,24 @@
+
+<div class="banner-image">
+    <a href="<%- marketingUrl %>" class="card-link">
+        <img alt-text="<%- gettext(name)%>" src="" />
+    </a>
+</div>
+<div class="text-section">
+    <div class="meta-info">
+        <div class="organization" aria-hidden="true">
```

Comment:
We agreed that if they were all within the ```<a>``` tag it would be too much information, but because only the program name is inside the tag it will be more easily read by screenreaders (i.e. as 3 separate elements instead of a single string).